components/camel-elasticsearch-rest/src/main/java/org/apache/camel/component/elasticsearch/ElasticsearchConfiguration.java

```diff
@@ -51,6 +51,10 @@
     private boolean disconnect;
     @UriParam(defaultValue = "false")
     private boolean enableSSL;
+    @UriParam(defaultValue = "false")
+    private boolean useScroll;
+    @UriParam(defaultValue = "" + ElasticsearchConstants.DEFAULT_SCROLL_KEEP_ALIVE_MS)
+    private int scrollKeepAliveMs;
```

Comment:
If you declare the default scroll keep alive as default Value you have to set on the variable too.

Reply:
Ok, I just pushed a fix for this and code style.
Thank you @oscerd.